crates/net/src/websocket/futures.rs

```diff
@@ -238,6 +238,20 @@ impl WebSocket {
     }
 }
 
+impl TryFrom<web_sys::WebSocket> for WebSocket {
+    type Error = JsError;
+
+    fn try_from(ws: web_sys::WebSocket) -> Result<Self, Self::Error> {
+        Self::setup(Ok(ws))
+    }
+}
+
+impl AsRef<web_sys::WebSocket> for WebSocket {
+    fn as_ref(&self) -> &web_sys::WebSocket {
+        &self.ws
+    }
+}
+
```

Comment:
Please remove this impl. We don't want to allow such easy access to the inner WebSocket instance. See the discussion at https://github.com/rustwasm/gloo/pull/337 for more details about why that is the case.